docs: Refine language in Oracle integration requirements (#21263)
newrelic/docs-website: src/content/docs/infrastructure/host-integrations/host-integrations-list/oracle-database-monitoring-integration.mdx

Before installing the integration, make sure that you meet the following requirements:

> * A Linux distribution that is [supported by the New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure) . Note that RHEL, CentOS, and OEL versions prior to 7 are not supported.

Reviewer: ```suggestion
* A Linux distribution that is [supported by the New Relic infrastructure agent](/docs/infrastructure/new-relic-infrastructure/getting-started/compatibility-requirements-new-relic-infrastructure).
    
    <Callout variant="tip" title="NOTE">
        RHEL, CentOS, and OEL versions prior to 7 are not supported.
    </Callout>
```

> * The [New Relic infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic) must be installed. [Install the infrastructure agent].

Reviewer: ```suggestion
* [Install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic).
```

> * The [Oracle Instant Client](http://www.Oracle.com/technetwork/database/database-technologies/instant-client/downloads/index.html) must be installed on the same host where the infrastructure agent is running.

Reviewer: ```suggestion
* Install the [Oracle Instant Client](http://www.Oracle.com/technetwork/database/database-technologies/instant-client/downloads/index.html) on the same host where the infrastructure agent is running.
```

> * The `ORACLE_HOME` environment variable must be configured correctly for the root user, pointing to the appropriate [Oracle database directory](https://docs.oracle.com/database/121/ADMQS/GUID-EC18C4A6-3BA5-4C14-9D76-B0DD62FEFFF2.htm#ADMQS12369) .

Reviewer: ```suggestion
* Configure the `ORACLE_HOME` environment variable for the root user to refer to the correct [Oracle database directory](https://docs.oracle.com/database/121/ADMQS/GUID-EC18C4A6-3BA5-4C14-9D76-B0DD62FEFFF2.htm#ADMQS12369).
```

> * An Oracle database user with [both <DNT>**CONNECT**</DNT> and <DNT>**SELECT**</DNT> privileges](#users-privileges) on the required global views.

Reviewer 2: ```suggestion
* An Oracle database user with [both <DNT>**CONNECT**</DNT> and <DNT>**SELECT**</DNT> privileges](#enable-instance) on the required global views.
```
